data/json/mapgen/grocery1.json

```diff
@@ -4,9 +4,12 @@
     "type": "item_group",
     "items": [
       [ "bread", 65 ],
+      [ "cake2", 10 ],
```

Comment:
side note: why cakes are priceless?

Reply:
@illi-kun I don't know. DracoGriffin's idea seems to be so.